### DISCUSSION TITLE
Styles don’t apply to tippy.js popup

### DISCUSSION THREAD

**Discussion by heymartinadams** [ueberdosis/tiptap#5302] [General Questions] [answered]
For some reason, none of the `bg-white dark:bg-black` styles apply to any tippy.js popups. Might anyone know why?

![CleanShot 2024-07-01 at 15 53 37@2x](https://github.com/ueberdosis/tiptap/assets/11673745/9f3043a9-e6ed-4827-9f45-3ca857d43964)

![CleanShot 2024-07-01 at 15 57 26@2x](https://github.com/ueberdosis/tiptap/assets/11673745/c4b33b0e-bc39-4a18-8e5b-360c0e335e5e)

**Comment by nperez0111** [comment, marked as answer]
You would have to see what is even making it transparent in the first place. Either way. Not really an issue with tiptap
